### ISSUE TITLE
2.0 isn't coloring/dinstinctint template literals  from strings

### ISSUE THREAD

**Issue by JavierPiedra** [sdras/night-owl-vscode-theme#274]
**Please make sure you're using the latest version of the plugin before submitting an issue**
![image](https://user-images.githubusercontent.com/42880686/96072021-f08b9800-0e68-11eb-9b05-c2cdcd0bc513.png)

**Screenshots**
If applicable, add screenshots to help explain your problem, it helps us debug faster.

If you are submitting a bug report in a language not currently in demo folder, a PR to this folder with the language helps us track it down.

Thanks!

**Comment by sdras** [comment]
well, hm... someone else asked that they be the same color. I have to think of what to do now. I think I agree with you, let me use the new version a bit and see.

**Comment by ivansvlv** [comment]
Do you think it's necessary or useful to colour them differently? Template literal produces a string so it's quite intuitive to consider it as a string. Besides, template literals usual usage is to interpolate values into the string, and I would argue that Night Owl has one of the best interpolation highlighting, thus making it pretty obvious what are you working with.

**Comment by JavierPiedra** [comment]
> Do you think it's necessary or useful to colour them differently? Template literal produces a string so it's quite intuitive to consider it as a string. Besides, template literals usual usage is to interpolate values into the string, and I would argue that Night Owl has one of the best interpolation highlighting, thus making it pretty obvious what are you working with.

Yes, it has been useful to me, specially to find it quickly. I notice the change.

**Comment by selensr** [comment]
This was really useful. I'd like to see that template literal coloring again. Actually, HTML tag and component colors are also changing to some green. It became really hard for reading and for eyes. 

![Screen Shot 2020-11-25 at 10 19 42](https://user-images.githubusercontent.com/45594532/100195564-96223480-2f08-11eb-8831-a082ce477c96.png)

**Comment by ivansvlv** [comment]
> This was really useful. I'd like to see that template literal coloring again. Actually, HTML tag and component colors are also changing to some green. It became really hard for reading and for eyes.
> 
> ![Screen Shot 2020-11-25 at 10 19 42](https://user-images.githubusercontent.com/45594532/100195564-96223480-2f08-11eb-8831-a082ce477c96.png)

Do you have this extension installed? - https://marketplace.visualstudio.com/items?itemName=mgmcdermott.vscode-language-babel

If yes, it might be causing your component colouring change.

**Comment by selensr** [comment]
> > This was really useful. I'd like to see that template literal coloring again. Actually, HTML tag and component colors are also changing to some green. It became really hard for reading and for eyes.
> > ![Screen Shot 2020-11-25 at 10 19 42](https://user-images.githubusercontent.com/45594532/100195564-96223480-2f08-11eb-8831-a082ce477c96.png)
> 
> Do you have this extension installed? - https://marketplace.visualstudio.com/items?itemName=mgmcdermott.vscode-language-babel
> 
> If yes, it might be causing your component colouring change.

Yes, it is indeed. Thank you a lot. Tags and texts back to normal.
